recipes/miniaudio/all/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.4)
+project(miniaudio C)
+
+include(GNUInstallDirs)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(MSVC OR MSVC90 OR MSVC10)
+    set(MSVC ON)
+    set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
+endif()
+
+set(SOURCE_SUBFOLDER ${CMAKE_CURRENT_SOURCE_DIR}/source_subfolder/extras)
+set(MINIAUDIO_LIBRARY miniaudio)
+
+set(MINIAUDIO_TARGETS ${MINIAUDIO_LIBRARY})
+
+add_library(${MINIAUDIO_LIBRARY} ${SOURCE_SUBFOLDER}/miniaudio_split/miniaudio.c)
+target_include_directories(${MINIAUDIO_LIBRARY} PRIVATE ${SOURCE_SUBFOLDER}/miniaudio_split/)
+
+set_target_properties(${MINIAUDIO_LIBRARY} PROPERTIES VERSION ${MINIAUDIO_VERSION_STRING})
+
+install(TARGETS ${MINIAUDIO_TARGETS}
+        BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}
```

```suggestion
add_library(miniaudio ${SOURCE_SUBFOLDER}/miniaudio_split/miniaudio.c)
target_include_directories(miniaudio PRIVATE ${SOURCE_SUBFOLDER}/miniaudio_split/)

set_target_properties(miniaudio PROPERTIES VERSION ${MINIAUDIO_VERSION_STRING})

install(TARGETS miniaudio
```

No need to obfuscate the target. Simpler is better. 

And BUNDLE DESTINATION is not necessary, it's a library.